pkg/aws/aws.go

```diff
@@ -44,7 +45,7 @@ var (
 	collectorSuccessDesc = prometheus.NewDesc(
 		prometheus.BuildFQName(cloudcost_exporter.ExporterName, subsystem, "collector_success"),
 		"Was the last scrape of the AWS metrics successful.",
```

Comment:
Can you clarify the description?
"Was the last scrape of the AWS metrics successful." sounds like a boolean that will flip between 0,1 (Gauge?), then shouldn't it be better for this to be a Counter instead?